presto-docs/pom.xml

```diff
@@ -62,6 +62,22 @@
                             </arguments>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>generate-thrift-idl</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                        <configuration>
+                            <mainClass>com.facebook.swift.generator.swift2thrift.Main</mainClass>
+                            <arguments>
+                                <argument>com.facebook.presto.connector.thrift.api.PrestoThriftService</argument>
+                                <argument>-recursive</argument>
+                                <argument>-out</argument>
+                                <argument>${project.basedir}/src/main/sphinx/include/PrestoThriftService.thrift</argument>
```

Comment:
We don't want to generate into the source directory (builds should never modify the source tree). Instead, we should either generate at runtime (and don't check in), or we should generate to a temporary file and verify the output is the same.